Add contact info and contributors to genomics/README.md (#51)
ScilifelabDataCentre/scilifelab-metadata-templates: genomics/README.md

In addition, the template includes SciLifeLab specific [organisational metadata fields](https://github.com/ScilifelabDataCentre/scilifelab-metadata-templates/blob/main/organisational_metadata_fields.yml) relevant for data provenance for the researcher as well as other metadata consumers at SciLifeLab. These can be omitted when submitting to public end repositories, such as ENA. 

## Contact
For __general questions__ about the _SciLifeLab Genomics Technical Metadata Template_, please contact data-management@scilifelab.se. 

For __specific questions regarding filled-in metadata files received as part of a data delivery from NGI__ as the data-producing unit, please contact support@ngisweden.se. 

## Contributors

> The genomics template has been developed in collaboration between the Genomics Platform, NBIS and SciLifeLab Data Centre. The following people have contributed with their expertise (in no particular order):

Reviewer: I guess they are in a particular order - by affiliation? So maybe we can say that instead?

Author: True.

Author: Fixed in https://github.com/ScilifelabDataCentre/scilifelab-metadata-templates/pull/51/commits/ec4878e2f53e1ad9130f098ce60ec4474bafeb59

> The genomics template has been developed in collaboration between the Genomics Platform, NBIS and SciLifeLab Data Centre. The following people have contributed with their expertise (in no particular order):

Author: ```suggestion
The genomics template has been developed in collaboration between the Genomics Platform, NBIS and SciLifeLab Data Centre. The following people have contributed with their expertise (ordered by affiliation):
```
